core/resharding_test.go

```diff
@@ -3,16 +3,47 @@ package core
 import (
 	"fmt"
 	"testing"
-)
 
-func TestFakeGetInitShardState(t *testing.T) {
-	ss := fakeGetInitShardState()
-	for i := range ss {
-		fmt.Printf("ShardID: %v, NodeList: %v\n", ss[i].ShardID, ss[i].NodeList)
-	}
-}
+	"github.com/harmony-one/harmony/core/types"
+	"github.com/stretchr/testify/assert"
+)
 
 func TestFakeNewNodeList(t *testing.T) {
 	nodeList := fakeNewNodeList(42)
 	fmt.Println("newNodeList: ", nodeList)
```

Comment:
why is it a test?

Reply:
Since it's random, I just by looking at the result, can be removed